client/stripe-utils/utils.js

```diff
@@ -489,3 +489,41 @@ export const initializeUPEAppearance = ( api, isBlockCheckout = 'false' ) => {
 export const getPaymentMethodName = ( paymentMethodType ) => {
 	return getPaymentMethodsConstants()[ paymentMethodType ] || 'stripe';
 };
+
+/**
+ * Hides payment method if it has set specific countries in the PHP class.
```

Comment:
```suggestion
 * Determines if the payment method is restricted to specific countries. 
```

I note that this is a copy of WooPayments function but just a minor suggestion. This function description suggests that it will hide the payment element when in fact it just returns a bool that can be used for that purpose.

Reply:
True, thanks! 🙂 Suggestion committed.